### PR TITLE
Create decisions trigger (#7)

### PR DESCRIPTION
**Pull request by parsec**
Reverts bytebot-chat/bytebot-party-pack#24

Ignore me being dumb, I'm fucking shite at git. This should work tho.

### CODE REVIEW

File: decisions.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"math/rand"
+	"strings"
+)
+
+func choose(nick string, msg string) string {
```
Comment (bbriggs) [inline decisions.go:8]:
```suggestion
func choose(nick, msg string) string {
```

File: utils.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/bytebot-chat/gateway-irc/model"
 	"github.com/go-redis/redis/v8"
 	"github.com/rs/zerolog/log"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```
Comment (bbriggs) [inline utils.go:11]: Why did we need to add that package namespace here?

**Review by bbriggs — CHANGES_REQUESTED** [review]
Simplify function declaration

File: utils.go (hunk 2 above)
Comment (parsec) [reply utils.go:11]: The linter in VS Code did it... i'm not really sure why.

File: decisions.go (hunk 1 above)
Comment (parsec) [reply decisions.go]: Addressed in https://github.com/bytebot-chat/bytebot-party-pack/pull/26/commits/16f9f02a1310ad43293a85c74411f8111ae72d2a

File: utils.go (hunk 2 above)
Comment (bbriggs) [reply utils.go:11]: Let's back it out until we have a reason.

Comment (parsec) [reply utils.go:11]: Sounds good. Will do that now.

**Review by bbriggs — APPROVED** [review]

**Review by parsec — COMMENTED** [review]
Blah, I made the changes requested. I'mma merge.

We do it live.

**Pull request merged**